lib/index.js

```diff
@@ -86,27 +87,33 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
+  var opts = this.options;
+  var eventProperties = track.properties();
+
+  if (eventProperties.revenue && opts.sendRevenueOnlyForOrderCompleted && track.event() === 'Order Completed') {
```

Comment:
hmm is there anyway we can make this more readable? Seems like you can refactor bunch of this `if/else` logic.